src/style.css

```diff
@@ -1 +1,109 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+:root {
+  --main-text-colot: #3D4E61;
+
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  color: var(--main-text-colot);
+}
+
+input {
```

Comment:
Set it in search-form__input

Reply:
I need to set color: var(--main-text-colot) in search-form__input ?

Reply:
do not style by tags. move all styles for input to .search-form__input selector.
box-sizing better to set in * selector